server/src/main/java/com/cloud/network/router/VpcVirtualNetworkApplianceManagerImpl.java

```diff
@@ -294,7 +295,23 @@ public boolean finalizeVirtualMachineProfile(final VirtualMachineProfile profile
             }
         }
 
-        return super.finalizeVirtualMachineProfile(profile, dest, context);
+        super.finalizeVirtualMachineProfile(profile, dest, context);
```

Comment:
just checking, we can safely ignore the return value here?

Reply:
yes @DaanHoogland 

it always return true in the parent class, 

https://github.com/apache/cloudstack/blob/37bb24f29e02480488312f1a0c8965c3543651d8/server/src/main/java/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java#L2153